### DISCUSSION TITLE
Portuguese Translation

### DISCUSSION THREAD

**Discussion by Lurkars** [Lurkars/gloomhavensecretariat#590] [Ideas]
Hi @Lurkars , I could work in a translation for Portuguese (Brazilian) - pt-BR for the application, based on [the official translated version of Gloomhaven published in Brazil by Galápagos Jogos](https://www.mundogalapagos.com.br/gloomhaven/produto/GLH001) that I own.

Since there are already three different language localizations here (de, fr and kr), would you know if I need to get any permission from Galápagos Jogos in Brazil to use the same terms they used in their translations? 
My concern is if I'd be breaking any copyright they have for the translations (if that's even possible), or if using their translations here would fall under fair use regardless.

_Originally posted by @ruanmed in https://github.com/Lurkars/gloomhavensecretariat/issues/104#issuecomment-2198437757_

**Comment by Lurkars** [comment]
@ruanmed

Currently the complete copyright topic is based on good will. Also see #265 .

So my 2 cents on this:
- I also don't have license agreements for any other translation
- I still hope, that if I get any copyright claim, that it will be okay for the holder, that i just remove the corresponding content since i am at least beeing noncommercial
- INAL, so you need to decide yourself what risks you want to take. Anyway i am sure, that first the repository/owner will get claimed and not the contributor

**Comment by raizdepi** [comment]
Hi @Lurkars, I saw your concern about copyrights, but could I translate it into pt-BR at my own risk and responsibility? Could you create the pt-BR language and add me as a contributor to this translation?

Thanks @raizdepi

**Reply by Lurkars** [reply]
Thanks, you should be able to translate via https://translate.gloomhaven-secretariat.de/projects/gloomhavensecretariat/-/pt/ you can login with your GitHub Account.
